hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/supervisor/monitor/ConsumersRuntimeMonitor.java

```diff
@@ -46,44 +50,61 @@ public ConsumersRuntimeMonitor(ConsumersSupervisor consumerSupervisor,
         hermesMetrics.registerGauge("consumers-workload.monitor.oversubscribed", () -> monitorMetrics.oversubscribed);
     }
 
-    public void checkCorrectness() {
-        Set<SubscriptionName> assignedSubscriptions = workloadSupervisor.assignedSubscriptions();
-        Set<SubscriptionName> runningSubscriptions = consumerSupervisor.runningConsumers();
-
-        monitorMetrics.assigned = assignedSubscriptions.size();
-        monitorMetrics.running = runningSubscriptions.size();
-
-        Set<SubscriptionName> missingSubscriptions = missing(assignedSubscriptions, runningSubscriptions);
-        monitorMetrics.missing = missingSubscriptions.size();
-        for (SubscriptionName subscriptionName : missingSubscriptions) {
+    public void log(Set<SubscriptionName> assigned,
```

Comment:
why public? also the ordering of methods is a bit off, `run()` should come first so it's easier to read